.github/workflows/build-with-bal-test-native.yml

```diff
@@ -0,0 +1,54 @@
+name: GraalVM Check
+
+on: 
+  schedule:
+    - cron: '30 18 * * *'
+  workflow_dispatch:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Set up JDK 11
+              uses: actions/setup-java@v3
+              with:
+                distribution: 'zulu'
```

Comment:
@keizer619 do we have any recommended jdk11 distribution for our workflows? is it `temurin`?

Reply:
Yes lets use `temurin` as we ship that